src/OctoshiftCLI.Tests/gei/Commands/MigrateOrg/MigrateOrgCommandTests.cs

```diff
@@ -21,6 +21,7 @@ public void Should_Have_Options()
             TestHelpers.VerifyCommandOption(command.Options, "github-source-pat", false);
             TestHelpers.VerifyCommandOption(command.Options, "github-target-pat", false);
             TestHelpers.VerifyCommandOption(command.Options, "verbose", false);
+            TestHelpers.VerifyCommandOption(command.Options, "target-api-url", false);
```

Comment:
A similar test to `It_Uses_Target_Api_Url_When_Provided` also needs to be added for this command too.

Reply:
okay will add tomorrow, thanks for the quick feedback!

Reply:
@ArinGhazarian added in https://github.com/github/gh-gei/pull/1214/commits/833931b72b39a5e1dd253973774f603db509a397, also note I was following the gei spec which originally added this flag, hence why I added test in handler to begin with but thanks for catching it 👍🏾  https://github.com/github/gh-gei/blob/main/src/OctoshiftCLI.Tests/gei/Commands/MigrateRepo/MigrateRepoCommandTests.cs

Reply:
Yeah a lot has changed in the CLI since then mostly because the underlying `System.CommandLine` nuget package pas early beta. Later on as the package became more mature, we changed the way we created the handlers to keep the handles' constructor very simple and easy to test and also keep the concerns as separate as possible so for each command we have the arg, the command and the handler, each one doing a specific thing. IMO the most interesting thing we did was to not inject factories into handlers and delegate the creation of a handler to the command via `BuildHanlder` and that's where all the creation logic goes.